hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -73,10 +73,17 @@ public class DatanodeConfiguration extends ReconfigurableConfig {
       "hdds.datanode.disk.check.min.gap";
   public static final String DISK_CHECK_TIMEOUT_KEY =
       "hdds.datanode.disk.check.timeout";
+
+  // Minimum space should be left on volume.
+  // Ex: If volume has 1000GB and minFreeSpace is configured as 10GB,
+  // In this case when availableSpace is 990GB or below, volume is assumed as full
```

Comment:
> `when availableSpace is 990GB or below, volume is assumed as full`

This doesn't sound right.  Volume is full if `availableSpace <= 10GB`, or `usedSpace >= 990GB`.